library/ssl_tls13_generic.c

```diff
@@ -1085,6 +1093,14 @@ int mbedtls_ssl_derive_traffic_keys( mbedtls_ssl_context *ssl, mbedtls_ssl_key_s
         return( ret );
     }
 
+    /*
+     * Export server handshake traffic secret
+     */
+#if defined(MBEDTLS_SSL_EXPORT_KEYS)
+    if( ssl->conf->f_export_secret != NULL )
```

Comment:
Same here: Overly long next line, and once that's broken, the branch requires `{ ... }` as per style convention.